src/contrib/cluster/Akka.Cluster.Sharding/Internal/EventSourcedRememberEntitiesProvider.cs

```diff
@@ -29,7 +30,15 @@ public EventSourcedRememberEntitiesProvider(string typeName, ClusterShardingSett
         /// <returns></returns>
         public Props ShardStoreProps(string shardId)
         {
-            return EventSourcedRememberEntitiesShardStore.Props(TypeName, shardId, Settings);
+            var backoffOptions = Backoff.OnStop(
```

Comment:
Wraps the `ShardStore` inside a `BackoffSupervisor` - uses the same tuning parameters as the shard itself.